services-isochrone/src/test/java/com/mapbox/api/isochrone/IsochroneTestUtils.java

```diff
@@ -0,0 +1,50 @@
+package com.mapbox.api.isochrone;
+
+import com.mapbox.core.TestUtils;
+
+import okhttp3.HttpUrl;
+import okhttp3.mockwebserver.Dispatcher;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+
+import org.hamcrest.junit.ExpectedException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+
+import java.io.IOException;
+
+public class IsochroneTestUtils extends TestUtils {
+    protected static final String ISOCHRONE_WITH_POLYGONS_VALID = "isochroneWithPolygons.json";
+    protected static final String ISOCHRONE_NO_POLYGONS_VALID = "isochroneNoPolygons.json";
```

Comment:
This one does not seem to be used.
It looks like all the tests will be using : ISOCHRONE_WITH_POLYGONS_VALID